PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+JIRA: https://jira.dev.bbc.co.uk/browse/YOUR-TICKET
+
+_PR changes summary to go here._
+
+- [ ] JIRA ticket added
+- [ ] Pull request URL added to JIRA ticket
+- [ ] Up-to-date with latest (`git pull --rebase origin latest`)
+- [ ] Tested locally `npm run dev` runs, http://localhost:3000/ renders a page
```

Comment:
It would be cleaner if all steps were consistent in structure. e.g. `[step desc] - [script]`, or something like that